easybuild/easyconfigs/r/Rosetta/Rosetta-3.5-ictce-4.1.13.eb

```diff
@@ -13,4 +13,8 @@ sources = ['%(namelower)s%(version)s_bundles.tgz']
 
 builddependencies = [('SCons', '2.3.0', '-Python-2.7.3')]
 
+modextravars = {
+    'ROSETTA3_DB': '$root/rosetta_database',
```

Comment:
ok, too complicated to include as a sanity check command then, I guess...

but, we do have an easyblock for Rosetta, so how about fixing this problem once in a single location (i.e. the easyblock), and then forget about it?

are you up for that, @Bart-CER?